pyetrade/accounts.py

```diff
@@ -215,16 +277,20 @@ def list_transactions(
         )
 
         payload = {
-            "startDate": start_date,
-            "endDate": end_date,
+            "startDate": start_date.date().strftime("%m%d%Y")
+            if start_date
+            else start_date,
+            "endDate": end_date.date().strftime("%m%d%Y") if end_date else end_date,

```

Comment:
This statement looks a little funny. I understand `end_date` would be `None` if it didn't hit the `if` statement but I think it should just be `else None` just so it is more explicit and clear.

Reply:
Ya, looking back, i agree, leaving it like that would only confuse someone who isn't familiar with the code. I'll make the change.

Reply:
See latest commits